client/www/scripts/modules/tracing/tracing.viz.module.js

```diff
@@ -27993,7 +27993,8 @@ module.exports={
     "shasum": "58b1f9c9458653aee3ebd9b0a5bc1a79a47c6bd2",
     "tarball": "http://ci.strongloop.com:4873/strong-trace-waterfalltransform/-/strong-trace-waterfalltransform-1.0.18.tgz"
   },
-  "_resolved": "http://ci.strongloop.com:4873/strong-trace-waterfalltransform/-/strong-trace-waterfalltransform-1.0.18.tgz"
+  "_resolved": "http://ci.strongloop.com:4873/strong-trace-waterfalltransform/-/strong-trace-waterfalltransform-1.0.18.tgz",
+  "readme": "ERROR: No README data found!"
```

Comment:
is this intentional or a byproduct of the other changes?
you should revert this file is you didn't intend this update.